### DISCUSSION TITLE
When to use copytocontainer and copyfromcontainer?

### DISCUSSION THREAD

**Discussion by dazz397** [markshust/docker-magento#993] [Q&A]
I have started development using docker magento and I am confused on when to use copy to and from container. When I edit code in vendor or app it seems to sync to the container but if I do a "bin/composer update --no-dev" and it removes the development packages in docker container but it keeps them on my local drive. Running copy to or from will not remove them. Do I need to delete my local files and do a copyfromcontainer -all? Is there any documentation on what files are synced automatically and what needs to be manually synced?

**Comment by YevhenZvieriev** [comment]
Hi, @dazz397 

You can check compose.dev.yaml
